find_me_a_sandwich/app/models/menu_item.rb

```diff
@@ -1,5 +1,6 @@
 class MenuItem < ActiveRecord::Base
   belongs_to :menu
+  belongs_to :favorite
```

Comment:
You don't need this relationship.  If you really want it, it should be `has_many :favorites`